mautic/mautic
# vTiger: Company name support

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
One of previous Mautic version divided integrations fields for contact/company.
This PR added support for Company name and therefor add Company Mappings tab.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Install vTiger plugin to Mautic
2. Contact mapping tab  is without Company field in Integration fields column


#### Steps to test this PR:
1. Apply PR
2. See Company mapping tab and Company field in Integrations tab

![image](https://cloud.githubusercontent.com/assets/462477/26331732/c62c9ea8-3f52-11e7-91cf-203c9395235c.png)
